README.md

```diff
@@ -123,6 +112,12 @@ Set up your computer
 
 4. Ensure that $DART_SDK is set to the path of your Dart SDK and 'adb'
    (inside 'platform-tools' in the android sdk) is in your $PATH.
+   
+For Windows users
+-----------------
+
+You will need to install Python 2.7.9 on your computer. Go on the Python site and download Python 2.7.9 : https://www.python.org/downloads/ .
```

Comment:
I think we can state this simpler by just saying that sky_tool requires python 2.